btf/types.go

```diff
@@ -514,6 +514,13 @@ type VarSecinfo struct {
 	Size   uint32
 }
 
+func (vsi *VarSecinfo) TypeName() string {
```

Comment:
Seems a bit confusing, since `TypeName` is part of `Type` but `VarSecInfo` doesn't implement the latter. Also missing context, commit message?

Reply:
Reading through the rest of the PR I still don't know where this is used.